fabtests/multinode/include/core.h

```diff
@@ -44,21 +44,35 @@
 
 #define PM_DEFAULT_OOB_PORT (8228)
 
+enum multi_xfer{
+	msg = 0,
+	rma = 1,
+};
+
+struct multi_xfer_method {
+		char* name;
+		int (*send)();
+		int (*recv)();
+		int (*wait)();
+};
+
 struct pm_job_info {
 	size_t		my_rank;
 	size_t		num_ranks;
 	int		sock;
 	int		*clients; //only valid for server
+	struct fi_rma_iov 	*multi_iovs;
 
 	struct sockaddr_storage oob_server_addr;
 	size_t 		server_addr_len;
 	void		*names;
 	size_t		name_len;
 	fi_addr_t	*fi_addrs;
+	enum multi_xfer caps;
```

Comment:
OFI defines caps as a bit mask.  Having this be an enum is expected.  Please rename to transfer type or something more descriptive.